src/ml_board/backend/dummy_server/dummy_server.py

```diff
@@ -136,27 +122,29 @@ def _on_leave(self):
 # main (starting point)
 
 
+params = ["path", "line_count", "port", "delay", "cors_ports"]
+# sys.argv EXAMPLE: path=event_storage.log line_count=500 port=7000 delay=0.1 cors_ports=3000,7000,8080
 if __name__ == '__main__':
 
-    # make sure the user input all arg_keys
-    input_len = len(sys.argv)
-    arg_keys = ['program_name', 'log_file_path', 'how_many_lines']
-    if input_len != len(arg_keys):
-        print(f"missing args: {arg_keys[input_len:]}")
-        print("how_many_lines could be -1 for the entire file!")
+    sys.argv.pop(0)
+    args = dict([arg.split('=') for arg in sys.argv])
+    print(f"{args}\n{'Valid parameters ✔' if all(arg in params for arg in args) else 'Invalid parameters ✗'}")
```

Comment:
no special characters in code only ascii

Reply:
Done, removed them!